src/v2/guide/migration.md

```diff
@@ -173,68 +173,68 @@ mounted: function () {
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> para encontrar ejemplos del orden de argumentos obsoleto. Tenga en cuenta que si nombra su argumento de índice con algún nombre inusual como <code>position</code> o <code>num</code>, el asistente no los encontrará.</p>
 </div>
 {% endraw %}
 
-### `v-for` Argument Order for Objects <sup>changed</sup>
+### `v-for` Orden de argumento para objetos <sup>cambiado</sup>
 
-When including a `key`, the argument order for objects used to be `(key, value)`. It is now `(value, key)` to be more consistent with common object iterators such as lodash's.
+Cuando se incluye una `clave`, el orden de los argumentos para los objetos solía ser `(clave, valor)`. Ahora es `(valor, clave)` para ser más consistente con iteradores de objetos comunes como lodash.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of the obsolete argument order. Note that if you name your key arguments something like <code>name</code> or <code>property</code>, the helper will not flag them.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper"> asistente de migración</a> para encontrar ejemplos del orden de argumento obsoleto. Tenga en cuenta que si nombra sus argumentos clave como <code>name</code> o <code>property</code>, el asistente no los encontrará.</p>
 </div>
 {% endraw %}
 
-### `$index` and `$key` <sup>removed</sup>
+### `$index` y `$key` <sup>removido</sup>
 
-The implicitly assigned `$index` and `$key` variables have been removed in favor of explicitly defining them in `v-for`. This makes the code easier to read for developers less experienced with Vue and also results in much clearer behavior when dealing with nested loops.
+Las variables implícitamente asignadas `$index` y `$key` se han eliminado a favor de definirlas explícitamente en `v-for`. Esto hace que el código sea más fácil de leer para los desarrolladores con menos experiencia en Vue y también da como resultado un comportamiento mucho más claro cuando se trata de bucles anidados.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of these removed variables. If you miss any, you should also see <strong>console errors</strong> such as: <code>Uncaught ReferenceError: $index is not defined</code></p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper"> asistente de migración</a> para encontrar ejemplos de estas variables removidas en su código. Si usted se pierde de alguna, también debería ver los <strong>errores de consola</strong> como: <code>Uncaught ReferenceError: $index is not defined</code></p>
```

Comment:
I think this will render two spaces, let's remove the one between `Ejecute el` and `asistente de migración`

```suggestion
  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> para encontrar ejemplos de estas variables removidas en su código. Si usted se pierde de alguna, también debería ver los <strong>errores de consola</strong> como: <code>Uncaught ReferenceError: $index is not defined</code></p>
```